File: app/src/main/java/com/nandy/contacts/mvp/view/ContactDetailsView.java
package com.nandy.contacts.mvp.view;

import android.graphics.Bitmap;

import com.nandy.contacts.model.Address;
import com.nandy.contacts.model.Email;
import com.nandy.contacts.model.Phone;
import com.nandy.contacts.mvp.BaseView;
import com.nandy.contacts.mvp.presenter.ContactDetailsPresenter;

import java.util.List;

/**
 * Created by yana on 30.11.17.
 */

public interface ContactDetailsView extends BaseView<ContactDetailsPresenter>{

    void setPhones(List<Phone> phones);

    void setEmails(List<Email> emails);

    void setAddresses(List<Address> addresses);

    void setName(String name);

    void setPhoto(Bitmap bitmap);

    void unbindViews();
}

File: app/src/main/java/com/nandy/contacts/mvp/view/ContactDetailsView.kt
package com.nandy.contacts.mvp.view

import android.graphics.Bitmap
import com.nandy.contacts.model.Address
import com.nandy.contacts.model.Email
import com.nandy.contacts.model.Phone
import com.nandy.contacts.mvp.BaseView
import com.nandy.contacts.mvp.presenter.ContactDetailsPresenter

/**
 * Created by yana on 03.12.17.
 */
interface ContactDetailsView: BaseView<ContactDetailsPresenter>{

    fun setPhones(phones: MutableList<Phone>)

    fun setEmails(emails: MutableList<Email>)

    fun setAddresses(addresses: MutableList<Address>)

    fun setName(name: String)

    fun setPhoto(bitmap: Bitmap?)

    fun unbindViews()
}